src/networking.c

```diff
@@ -4548,9 +4548,9 @@ static void pauseClientsByClient(mstime_t endTime, int isPauseClientAll) {
  * The function always succeed, even if there is already a pause in progress.
  * The new paused_actions of a given 'purpose' will override the old ones and
  * end time will be updated if new end time is bigger than currently configured */
-void pauseActions(pause_purpose purpose, mstime_t end, uint32_t actions) {
+void pauseActions(pause_purpose purpose, mstime_t end, uint32_t actions_bitmask) {
```

Comment:
Is it a bitmask?

I think it's a bitmap. A mask is used for masking, setting some bits to 1 or 0.

Maybe this variable rename is not necessary?

Reply:
I changed it because it is named like this in server.h
```
void pauseActions(pause_purpose purpose, mstime_t end, uint32_t actions_bitmask);
```

Reply:
i rename it back to actions.

Reply:
Ah, ok, maybe bitmask can be ok too... Not important..